### PR TITLE
Add dynamo-release

### PR DESCRIPTION
**Pull request by Ukyeon**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Sichao25** [comment]
@Xiaojieqiu Hi xiaojie, are you willing to be the maintainer of this recipe?

**Comment by Xiaojieqiu** [comment]
yes, I would love to be.

**Comment by Sichao25** [comment]
@conda-forge-admin please ping conda-forge/help-python, ready for review!

File: recipes/dynamo-release/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aristoteleo/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4b9e0b9721802b0b7baa05a0651518958d512660703ca0d62a5140a7689f3d22
+
+build:
```
Comment (BastianZim) [inline recipes/dynamo-release/meta.yaml:12]: Can this be converted to a noarch recipe? https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/dynamo-release/meta.yaml (hunk 1 above)
Comment (Sichao25) [reply recipes/dynamo-release/meta.yaml:12]: Thanks for the suggestion. I have added the noarch build.

File: recipes/dynamo-release/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  summary: 'Inclusive model of expression dynamics with metabolic labeling based scRNA-seq'
+  description: |
+    Inclusive model of expression dynamics with metabolic labeling based scRNA-seq in silico perturbation predictions.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - Xiaojieqiu
```
Comment (BastianZim) [inline recipes/dynamo-release/meta.yaml:74]: @Ukyeon and @Sichao25 This means that you won't be a maintainer, is that correct?

File: recipes/dynamo-release/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+requirements:
+  build:
+    - python >=3.7
+  host:
+    - python >=3.7
+    - numpy 1.23.*
+    - markdown 3.3.*
+    - setuptools
+    - pip
```
Comment (BastianZim) [inline recipes/dynamo-release/meta.yaml:25]: Do you need all of these? Upstream only specifies python and nothing else?

File: recipes/dynamo-release/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+#  commands:
+#    - pip check # fails probably because package python-igraph has a different name in pip
+
+about:
+  home: https://github.com/aristoteleo/dynamo-release
+  summary: 'Inclusive model of expression dynamics with metabolic labeling based scRNA-seq'
+  description: |
+    Inclusive model of expression dynamics with metabolic labeling based scRNA-seq in silico perturbation predictions.
+  license: MIT
+  license_family: MIT
```
Comment (BastianZim) [inline recipes/dynamo-release/meta.yaml:69]: Upstream mentions a different license: https://github.com/aristoteleo/dynamo-release/blob/master/LICENSE

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

File: recipes/dynamo-release/meta.yaml (hunk 4 above)
Comment (Sichao25) [reply recipes/dynamo-release/meta.yaml]: Thanks for reminding. I have updated to the corrected one, BSD 3-Clause.

File: recipes/dynamo-release/meta.yaml (hunk 3 above)
Comment (Sichao25) [reply recipes/dynamo-release/meta.yaml]: I think we need all of these in "host". I have removed the "build" section.

**Pull request merged**